src/globals/mixins.scss

```diff
@@ -368,13 +368,15 @@
   border-radius: 5px;
   border: 1px solid $colour-black;
   background-color: $colour-black;
-  filter: drop-shadow(0px 8px 20px rgba(0, 0, 0, 0.22));
   color: $colour-white;
   display: inline-block;
   text-decoration: none;
   text-align: center;
   transition: all ease-in-out 0.5s;
   cursor: pointer;
+  &.shadow-true {
+    filter: drop-shadow(0px 8px 20px rgba(0, 0, 0, 0.22));
```

Comment:
@tomwpaterson-shape  We are seeing major performance problems in desktop Safari on big pages which @alihejazi thinks is down to this filter. Can we please remove it + the controlling prop across the board for now?